cluster-provision/k8s/1.17/provision.sh

```diff
@@ -10,7 +10,6 @@ xfs_growfs -d /
 cni_manifest="/tmp/cni.yaml"
 
 setenforce 0
```

Comment:
should we be removing `setenforce 0`  here as well?

Reply:
ah, good point, I'll remove it.